main.tf

```diff
@@ -98,11 +98,12 @@ resource "google_cloudfunctions2_function" "webhook" {
     available_memory      = "1G"
     service_account_email = google_service_account.webhook.email
     environment_variables = {
-      OUTPUT_BUCKET   = google_storage_bucket.main.name
-      DOCAI_PROCESSOR = google_document_ai_processor.ocr.id
-      DOCAI_LOCATION  = google_document_ai_processor.ocr.location
-      BQ_DATASET      = google_bigquery_dataset.main.dataset_id
-      BQ_TABLE        = google_bigquery_table.main.table_id
+      OUTPUT_BUCKET    = google_storage_bucket.main.name
+      DOCAI_PROCESSOR  = google_document_ai_processor.ocr.id
+      DOCAI_LOCATION   = google_document_ai_processor.ocr.location
+      BQ_DATASET       = google_bigquery_dataset.main.dataset_id
+      BQ_TABLE         = google_bigquery_table.main.table_id
+      LOG_EXECUTION_ID = true
```

Comment:
Reason I've added this `LOG_EXECUTION_ID = true` line:
The verify step in our integration tests were failing:
```
    module.simple.google_cloudfunctions2_function.webhook will be updated in-place
    "google_cloudfunctions2_function" "webhook" {
        service_config {
            environment_variables            = {
                "LOG_EXECUTION_ID"  = "true" -> null
…
Plan: 0 to add, 1 to change, 0 to destroy.
```

It sounds like Google Cloud (under the hood) sets this LOG_EXECUTION_ID environment variable to true — after we terraform apply. So subsequent "terraform apply"s think the Cloud Function needs to be updated since, in our Terraform definition, the value is null (not set).